manifest-dev.yaml

```diff
@@ -0,0 +1,12 @@
+---
+applications:
+- name: funding-service-design-assessment-store-dev
```

Comment:
this name is wrong

Reply:
What should it be? I think I've missed something.

Reply:
app is `funding-service-design-account-store` but this says `funding-service-design-assessment-store-dev`